# Experimental Techniques

| Label | Alternative Terms | ID | ontology ID |
|-------|-------------------|----|-------------|
| droplet-based single-cell RNA sequencing assay | Dropseq | 999bc73a-b4dc-4824-929a-fe61524368d2 | OBI:0003376 |
| fluorescence in-situ hybridization assay | FISH | 9e9d2c80-3430-4404-9c12-28ee50f5dae8 | OBI:0003094 |
| fluorescence micro-optical sectioning tomography assay | fMOST | 39ab2cda-9253-4790-9df0-996e5a24fe5d | BITO:0000001 |
| multiplexed error-robust fluorescence in situ hybridization assay | MERFISH | d7619c9a-c7e0-4893-8612-89092c376410 | BITO:0000002 |
| neuron morphology assay |  | 9c79b356-e08c-4ce1-b4d4-1bef7c7c797b | OBI:0003124 |
| mosaicism with repeat frameshift genetic sparse labeling | MORF genetic sparse labeling | 303590a2-dcde-48db-a61d-caeb7dbea580 | OBI:0003125 |
| serial two-photon tomography assay | STPT | e01c3bcb-6221-42b2-b337-09f518451b8f | BITO:0000003 |
| mouselight assay |  | 9bf571f0-50cd-4281-88bf-06003a5e67ea | BITO:0000004 |
| multi-electrode extracellular electrophysiology assay |  | 3c77f922-a3eb-46db-82ca-1f4a19dd24ad | BITO:0000005 |
| neuron morphology reconstruction assay |  | 9c79b356-e08c-4ce1-b4d4-1bef7c7c797b | OBI:0003126 |
| patch clamp single-cell RNA sequencing assay | patch-seq, patch clamp sequencing assay | a9faecd5-8da1-4f3c-8769-d2ce3ccb8c92 | OBI:0003375  |
| single cell combinatorial indexing assay for transposase-accessessable chromatin using sequencing | sciATACseq | a6049840-8545-45ed-9707-6edb3964a02b | OBI:0003104  |
| single-cell combinatorial indexing RNA sequencing assay | sciRNAseq | ad19aefb-72d8-4192-b922-47d5c4f0ba9f | OBI:0003105  |
| sequential fluorescence in-situ hybridization assay | seqFISH | 0f744a0c-ce68-497c-a03b-6f7f720e627a | OBI:0003106 |
| slide-seq assay | Slide-seq  | f86fba71-22a6-434b-ba7a-bf62ea76d3ad | OBI:0003107 |
| single-nucleus chromatin accessibility and mRNA expression sequencing assay | SNAREseq, SNARE-seq | 355e85d2-8501-4142-9d7b-d91e5b66f801 | OBI:0003108 |
| single-nucleus RNA sequencing assay | single nuclei RNA sequencing, sNuc-seq, snRNA-seq | 095a266b-ece3-49ec-b19e-ec8819d64e43 | OBI:0003109 |
| decoding amplified targeted transcripts with fluorescence in situ hybridization assay | DARTFISH |  | OBI:0003095 |
| bulk assay for transposase-accessible chromatin using sequencing | bulk ATAC-seq |  | OBI:0003089 |
| single-cell ATAC-seq | scATAC-seq |  | OBI:0002764 |
| single-nucleus ATAC-seq | snATAC-seq |  | OBI:0002762 |
| single-cell RNA sequencing assay | scRNA-seq | | OBI:0002631 |
| retrograde tracing assay | retrograde tracing, retrograde labeling | 5ed06811-c247-42af-9ea6-9a4614888255 | BITO:0000006 |
| retrograde transsynaptic tracing assay |  | 635ff3f8-5a09-44d7-bb8a-495baec715b0 | BITO:0000007 |
| switching mechanism at the 5' end of RNA template sequencing assay | SMARTseq, SMART seq | | OBI:0003378 |
| switching mechanism at the 5' end of RNA template sequencing 2 assay | SMARTseq2 | 5d00f8c4-5163-47e1-87cd-bc11c6e53f9c | OBI:0003379 |
| single-nucleus chromatin accessibility and mRNA expression sequencing 2 assay | SNARE-seq2, SNARE-seq2 assay | 355e85d2-8501-4142-9d7b-d91e5b66f801 | OBI:0003377 |
| switching mechanism at the 5' end of RNA template sequencing assay v4 | SMARTseq v4 | aa811d13-d900-4c5d-960e-c2ba5b756111 | BITO:0000008 |
| tracing the relationship between input and output study design | TRIO tracing | 23ca844a-e0f4-4e75-a4b7-29d482f9bc9d | BITO:0000009 |
| lightsheet fluorescence microscopy assay | lightsheet | 952c036b-88b3-41d9-823a-05ee645b4ecb | OBI:0003098 |
| cre-dependent anterograde tracing assay | | 3b24f0f6-bcbb-4060-8450-033a41af44e0 | BITO:0000010 |
| anterograde tracing assay | | 7dfb74fa-0fe2-45cc-91b6-fd0627d75b5d | BITO:0000011 |
| optical coherence tomography assay | OCT | 3f63e583-826d-43b5-9ed3-1b3fc068b9b5 | BITO:0000012 |
| volumetric imaging with synchronized on-the-fly oblique-scan and readout assay | VISoR, VISoR assay | 7504f4e2-ef04-429a-8705-5b57af859adc | BITO:0000013 |
| composite in situ imaging | CISI | 8c21802a-b576-40fd-92cf-f335b1bf1342 | BITO:0000014 |
| 3-level single-cell combinatorial indexing RNA sequencing assay | sci-RNA-seq3, 3-level sci-RNA-seq | ad19aefb-72d8-4192-b922-47d5c4f0ba9f | BITO:0000015 |
| 3-level single-cell combinatorial indexing ATAC sequencing assay | sci-ATAC-seq3, 3-level sci-ATAC-seq | a6049840-8545-45ed-9707-6edb3964a02b | BITO:0000016 |
| 10X chromium ATAC-seq assay | 10X chromium ATAC-seq\|10X chromium ATAC sequencing assay | | BITO:0000017 |
| SHARE-seq assay | SHARE-seq | de0f17bf-7610-41d0-a1ee-6cfcf06fcfa1 | BITO:0000018 |
| single cell multiome ATAC + gene expression assay | single cell multiome ATAC + gene expression | | BITO:0000019 |
| double in vitro transcription with absoluted counts sequencing assay | DIVA-seq | | BITO:0000020 |
| oblique light sheet tomography assay | OLST | 4da3d706-b02c-4e87-baf6-ba9cd775fc0d | BITO:0000021 |
| cyclic single-molecule fluorescence in situ hybridization assay | osmFISH | f62b09ad-1608-4160-a712-5c5d20b66981 | BITO:0000022 |
| SNARE-seq2 (ATAC-seq) | | | BITO:0000023 |
| SNARE-seq2 (RNA-seq) | | | BITO:0000024 |
| nanopore DNA sequencing assay | | Oxford Nanopore long-read sequencing | BITO:0000025 |
| single cell isoform RNA sequencing assay | ScISOr-seq | | BITO:0000026 |
| single nucleus methyl-3C sequencing assay | sn-m3c-seq | 34ab1458-fafb-41be-a9b8-8641133e71a5 | BITO:0000027 |
| single nucleus methyl-C sequencing assay version 2 | snmC-seq2 | | BITO:0000028 |
| mC-seq3 | | | BITO:0000029 |
| single-cell combinatorial indexing with WGBS methylation analysis version 2 | sci-METv2 | 4874d71f-f84f-461d-9f99-706105f55fc7 | BITO:0000030 |
| multiplex immunoassay | | ba9b2f4d-5794-4276-8bf2-9bcc33a19b24 | BITO:0000031 |
| immunoassay | | | BITO:0000032 |
| calcium imaging assay | Ca2+ imaging, Ca imaging | 0a3747af-571a-4277-9079-6bc7335c269b | BITO:0000033 |
| cellular assay | | | BITO:0000034 |
| human brain optimized light sheet microscopy assay | HOLiS | | BITO:0000035 |
| temp-O sequencing assay | temp-O seq | 3603db94-a6f1-4459-8dd0-b4724757b1a9 | BITO:0000036 |
| x-ray holographic nano-tomography assay | XHN | | BITO:0000037 |
| Pacific Bioscience long-read sequencing assay | | 04e4eece-1a56-40e6-a213-43e80ad1af06 | BITO:0000038 |
| enhancer virus labelling assay | enhancer virus labeling | 4aacc6ca-2042-4149-ac0f-a60222784fde | BITO:0000039 |
| paired-tag assay | | 968ef9e0-0a91-4a04-9d94-1f522a67f873 | BITO:0000040 |
| whole genome sequencing assay | WGS | 2bfee61e-4077-450b-a568-64c63cccbffb | OBI:0002117 |
| magnetic resonance imaging assay | MRI | 2b0361d3-6d3e-4e43-8e9f-e0b76eb64e24 | OBI:0002985  |
| DNA sequencing assay | nucleotide sequencing | fd41eb65-d7da-4afb-956b-0174f3547b22 | OBI:0000626 |
| DBiT-seq | | | BITO:0000041 |
| Perturb-seq | CRISP-seq, CROP-seq | | BITO:0000042 |
| SMARTSeqSC | | | BITO:0000043 |
| SmartSeq3 | | | BITO:0000044 |
| 10xV3.1 | | | BITO:0000045 |
| 10xV3.1_HT | | | BITO:0000046 |
| 10xMultiome;GEX | | | BITO:0000047 |
| 10xMultiome;ATAC | | | BITO:0000048 |
| 10xATAC_V2.0 | | | BITO:0000049 |
| 10XMultiome-CellHashing;GEX | | | BITO:0000050 |
| 10XMultiome-CellHashing;ATAC | | | BITO:0000051 |
| 10XMultiome-Cell Hashing;Barcode | | | BITO:0000052 |
| 10xV3.1_CellPlex;GEX | | | BITO:0000053 |
| 10xV3.1_CellPlex;Barcode | | | BITO:0000054 |
| 10xV3.1_HT_CellPlex;GEX | | | BITO:0000055 |
| 10xV3.1_HT_CellPlex;Barcode | | | BITO:0000056 |
| MethylC-Seq | | | BITO:0000057 |
| snmCT-seq | | | BITO:0000058 |
| scATAC-seq | | | BITO:0000059 |
| Slide-seq MERFISH | | | BITO:0000060 |
| whole brain MERFISH | | | BITO:0000061 |
| DBiT RNA-seq | | | BITO:0000062 |
| DBiT ATAC-seq | | | BITO:0000063 |
| Slide-tag - spatial | | | BITO:0000064 |
| Slide-tag - GEX | | | BITO:0000065 |
| 10X 5P V2 - GEX | | | BITO:0000066 |
| PIPseq T2 - 3P GEX | | | BITO:0000067 |
| PIPseq T20 - 3P GEX | | | BITO:0000068 |
| Slide-seq standard puck with 10X 5P v2 - GEX and spatial | | | BITO:0000069 |
